pages/index.tsx

```diff
@@ -1,13 +1,25 @@
-import type { NextPage } from "next";
 import Head from "next/head";
-import Image from "next/image";
 import { FC } from "react";
-import styles from "../styles/Home.module.css";
+
+import { FileList, Header } from "../components";
 
 const Home: FC = () => {
   return (
-    <div className={styles.container}>
-      <h1 className="text-3xl font-bold underline">Hello world!</h1>
+    <div>
+      <Head>
+        <title>Western Pennsylvania Electrical Employees</title>
+        <meta
+          name="description"
+          content="This is a list of documents for members of the Western Pennsylvania Electrical Employees"
```

Comment:
💬 I tend to prefer more direct language. But it's tomato/🍅 

```suggestion
          content="List of documents for members of the Western Pennsylvania Electrical Employees"
```